web/content/js/versions.js

```diff
@@ -212,7 +224,11 @@ const navVersions = [
             "path": "Installing_Proxy"
           },
           {
-            "title": "Upgrading Foreman",
+            "title": "Updating Katello",
+            "path": "Updating_Project"
+          },
+          {
+            "title": "Upgrading Katello",
```

Comment:
Can we be explicit what Upgrading is?
```suggestion
            "title": "Upgrading Katello to nightly",
```